eclair-core/src/main/scala/fr/acinq/eclair/wire/protocol/OfferCodecs.scala

```diff
@@ -130,16 +129,15 @@ object OfferCodecs {
   })
 
   private val paymentInfo: Codec[PaymentInfo] = (("fee_base_msat" | millisatoshi32) ::
-    ("fee_proportional_millionths" | tu32) ::
-    ("cltv_expiry_delta" | cltvExpiryDelta)).as[PaymentInfo]
+    ("fee_proportional_millionths" | uint32) ::
+    ("cltv_expiry_delta" | cltvExpiryDelta) ::
+    ("htlc_minimum_msat" | millisatoshi) ::
+    ("htlc_maximum_msat" | millisatoshi) ::
+    ("features" | variableSizeBytes(uint16, bytes).xmap[Features[Feature]](Features(_), _.toByteVector))).as[PaymentInfo]
```

Comment:
Can you reuse the `lengthPrefixedFeaturesCodec`?

```suggestion
    ("features" | lengthPrefixedFeaturesCodec).as[PaymentInfo]
```

Reply:
Done